docs/Makefile

```diff
@@ -4,7 +4,7 @@
 # You can set these variables from the command line.
 SPHINXOPTS    =
 SPHINXBUILD   = python -msphinx
-SPHINXPROJ    = minos_microservice_saga
+SPHINXPROJ    = minos_microservice_common
```

Comment:
I think this would be referred to the `minos-microservice-saga` repo, I am right?